modules/ROOT/pages/branded_ios_app/publishing_ios_app_10.adoc

```diff
@@ -0,0 +1,88 @@
+= Additional Server Configuration 
+:apple-universal-link-support-url: https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html
+:admin_manual: https://doc.owncloud.com/server/administration_manual/index.html
+:apple-app-site-association-file-url: https://developer.apple.com/documentation/security/password_autofill/setting_up_an_app_s_associated_domains
+:aliasmatch-uri: https://httpd.apache.org/docs/2.4/mod/mod_alias.html#aliasmatch
+:rewritecond-directive-uri: http://httpd.apache.org/docs/2.2/mod/mod_rewrite.html#RewriteCond
+:generate-ios-app-url: branded_ios_app/publishing_ios_app_7.adoc#generate-ios-app
+
+== Add Support for Apple Universal Links
```

Comment:
Proposal, add:
```
== Introduction

Support of Universal Links
When you support universal links, iOS users can tap a link to your website and get seamlessly redirected to
your  installed app without going through Safari. If your app isn’t installed, tapping a link to your website
opens your website in Safari. For more details see:
https://developer.apple.com/library/archive/documentation/General/Conceptual/AppSearch/UniversalLinks.html
```

Reply:
I'm not an iOS developer or guru, but this seems a little strange to me. Did you mean this as an admonition or aspect of supporting documentation for the "Add Support for Apple Universal Links" section on that page?

Reply:
yes, aspect of supporting documentation. helpful to understand the topic

Reply:
Thanks.

Reply:
Done.